### PR TITLE
feat: Auto-fill search input when pressing Ctrl + Shift + F

### PR DESCRIPTION
**Pull request by CactusPuppy**
Demonstration of PR in action:
firefox_cQiN33b8wG

### CODE REVIEW

**Comment by CactusPuppy** [comment]
For some reason, the `editorStates` store doesn't properly track when the user selects something.

File: app/javascript/src/components/editor/FindReplaceAll.svelte [hunk 1]
```diff
@@ -115,7 +115,11 @@
     if (event.ctrlKey && event.shiftKey && event.keyCode == 70) { // F key
       event.preventDefault()
       active = !active
-      if (active) focusInput()
+      if (active) {
+        focusInput()
+        const codeMirrorSelectedText = $editorStates[$currentItem.id].sliceDoc($editorStates[$currentItem.id].selection.main.from, $editorStates[$currentItem.id].selection.main.to)
```
Comment (Mitcheljager) [inline app/javascript/src/components/editor/FindReplaceAll.svelte:120]: Perhaps `$editorStates[$currentItem.id].selection.main.from, $editorStates[$currentItem.id].selection.main.to` could be more readable like
```suggestion
const [from, to] = $editorStates[$currentItem.id].selection.main
const codeMirrorSelectedText = $editorStates[$currentItem.id].sliceDoc(from, to)
```

**Review by Mitcheljager — CHANGES_REQUESTED** [review]
Very nice overall, nice and simple. One small suggestion, but it's also good as is

File: app/javascript/src/components/editor/FindReplaceAll.svelte (hunk 1 above)
Comment (CactusPuppy) [reply app/javascript/src/components/editor/FindReplaceAll.svelte:120]: Oooh, I like it!

**Pull request merged**